range.c

```diff
@@ -2192,6 +2218,76 @@ range_count(int argc, VALUE *argv, VALUE range)
  *   r.include?(Xs.new(5))    #=> true
  *   r.include?(Xs.new(7))    #=> false
  *
+ * == What's Here
+ *
+ * First, what's elsewhere. \Class \Range:
+ *
+ * - Inherits from {class Object}[Object.html#class-Object-label-What-27s+Here].
+ * - Includes {module Enumerable}[Enumerable.html#module-Enumerable-label-What-27s+Here],
+ *   which provides dozens of additional methods.
+ *
+ * Here, class \Range provides methods that are useful for:
+ *
+ * - {Creating a Range}[#class-Range-label-Methods+for+Creating+a+Range]
+ * - {Querying}[#class-Range-label-Methods+for+Querying]
+ * - {Searching}[#class-Range-label-Methods+for+Searching]
+ * - {Comparing}[#class-Range-label-Methods+for+Comparing]
+ * - {Iterating}[#class-Range-label-Methods+for+Iterating]
+ * - {Converting}[#class-Range-label-Methods+for+Converting]
+ * - {Handling JSON}[#class-Range-label-Methods+for+Handling+JSON]
+ *
+ * === Methods for Creating a \Range
+ *
+ * - ::new:: Returns a new range.
+ *
+ * === Methods for Querying
+ *
+ * - #begin:: Returns the begin value given for +self+.
+ * - #count:: Returns a count of elements in +self+: absolute,
+ *            based on a given argument, or based on a given block.
+ * - #cover?:: Returns whether a given object is within +self+.
+ * - #end:: Returns the end value given for +self+.
+ * - #exclude_end?:: Returns whether the end object is excluded.
+ * - #first:: Returns the first elements of +self+.
+ * - #hash:: Returns the integer hash code.
+ * - #include? (aliased as #member?):: Returns whether a given object
+ *                                     is an element of +self+.
+ * - #last:: Returns the last elements of +self+.
+ * - #max:: Returns the maximum values in +self+.
+ * - #min:: Returns the minimum values in +self+.
+ * - #minmax:: Returns the minimum and maximum values in +self+.
+ * - #size:: Returns the count of elements in +self+.
+ *
+ * === Methods for Searching
+ *
+ * - #bsearch:: Returns an element from +self+ selected by a binary search.
+ *
+ * === Methods for Comparing
+ *
+ * - {#==}[#method-i-3D-3D]:: Returns whether a given object is equal to +self+
+ *                            (uses #==).
+ * - #===:: Returns whether the given object is between the begin and end values.
+ * - #eql?:: Returns whether a given object is equal to +self+ (uses #eql?).
+ *
+ * === Methods for Iterating
+ *
+ * - #%:: Iterates over selected elements of +self+,
```

Comment:
All three entries have the same description, but slightly different behavior.  `step` should indicate it only yields nth elements (defaulting to 1).  `%` should mention it is the same as `step`, but requires an argument.

Reply:
Done..